examples/synthetics_mesh_example_test.go

```diff
@@ -13,94 +13,167 @@ import (
 	"text/tabwriter"
 	"time"
 
-	syntheticspb "github.com/kentik/api-schema-public/gen/go/kentik/synthetics/v202101beta1"
+	syntheticspb "github.com/kentik/api-schema-public/gen/go/kentik/synthetics/v202202"
+	"github.com/kentik/community_sdk_golang/kentikapi"
 	"github.com/stretchr/testify/assert"
 	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
-func TestGetMeshTestResultsExample(t *testing.T) {
+func TestDemonstrateSyntheticsNetworkMeshTestResults(t *testing.T) {
```

Comment:
The example has following output now:

```
Picked network_mesh test named "network_mesh_6x6" with ID 6232
Number of test results: 50
Latest test results time: 2022-04-01 09:00:00 +0000 UTC
Table cells contain ping latency and connection health in format: "current [ms] / rolling avg. [ms] / rolling stddev. [ms] / health"
.                             Cape Town, South Africa    Chicago, IL, United States    Osaka, Japan                 Zhangjiakou, China          Petah Tikva, Israel        Tallinn, Estonia
Cape Town, South Africa       [X]                        245 / 242 / 2 / healthy       381 / 381 / [X] / healthy    286 / 289 / 1 / healthy     219 / 219 / 2 / healthy    183 / 177 / 2 / healthy
Chicago, IL, United States    [X]                        [X]                           [X]                          226 / 220 / 8 / healthy     148 / 148 / 3 / healthy    124 / 124 / 0 / healthy
Osaka, Japan                  [X]                        187 / 236 / 13 / healthy      [X]                          136 / 128 / 34 / healthy    340 / 332 / 3 / healthy    290 / 286 / 2 / healthy
Zhangjiakou, China            [X]                        226 / 218 / 3 / healthy       [X]                          [X]                         317 / 298 / 8 / healthy    [X] / 254 / 2 / critical
Petah Tikva, Israel           [X]                        148 / 148 / 3 / healthy       [X]                          318 / 297 / 6 / healthy     [X]                        86 / 81 / 2 / healthy
Tallinn, Estonia              [X]                        124 / 124 / 0 / healthy       [X]                          [X]                         85 / 82 / 2 / healthy      [X]
```

Reply:
Why not print also packet loss?

Reply:
Because it requires more work to do that. Created ticket for that improvement: https://clkentik.atlassian.net/jira/software/c/projects/KNTK/boards/1?view=detail&selectedIssue=KNTK-403

Reply:
Done: https://github.com/kentik/community_sdk_golang/pull/94